apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -128,35 +128,25 @@ const themeObject = {
     },
     body1: {
       fontWeight: 400,
-      fontSize: '20px',
-      lineHeight: '150%',
-      letterSpacing: '-0.019em',
+      fontSize: '16px',
+      lineHeight: '20px',
+      letterSpacing: '0',
```

Comment:
This change is for the new 2fa page
TODO: Revert and see where we are currently using body1